api/db/affiliations.js

```diff
@@ -200,9 +201,27 @@ const updateAuthAffiliation = async ({
     throw new Error('User is editing their own affiliation');
   }
 
+  // Lookup role id of the user making the affiliation update
+  const { role_id: changedByRoleId } = await (transaction || db)(
```

Comment:
I think all of this information is set in req.user when they hit the endpoint, you might not need to look it up again here

Reply:
yes! writing that query was killing me it felt so inefficient. not sure how I didn't realize I could get it from req.user considering I had to get `req.user.stateId` for something else 😄